libs/api-interfaces/src/lib/aggregate-views/view-models/resources/audio-item/multilingual-text-item.interface.ts

```diff
@@ -0,0 +1,10 @@
+import { LanguageCode } from '../../multilingual-text';
+import { MultiLingualTextItemRole } from './multilingual-text-item-role.enum';
+
+export interface IMultlingualTextItem {
```

Comment:
Just a thought, would these possibly be versions?  Or is there some linguistic term for the different

Reply:
Well you could say "translations", but we want to mark the distinction between a free translation and a literal translation and a gloss of an individual word. While `item` is a bit generic, I think it's safe in this regard. 

Note that "versions" is not correct. It doesn't capture the one way relationships like "translation of", "glossed from", "prompt for", etc.

Reply:
Alternatively, we could call them all translations, and then have
gloss translation
free translation
prompt for translation
but I still don't think it makes sense to call the `original` a translation.

Reply:
I wonder if we could get some of Dr. Radney's time to help us nail down the terminology. One interesting thing is that the user will never know about the item directly. Along that line, I wonder if it makes sense to have separate
`GLOSS_RESOURCE` and `TRANSLATE_RESOURCE` commands.